static/js/ui_init.js

```diff
@@ -670,6 +671,13 @@ export function initialize_everything() {
     user_status_ui.initialize();
     fenced_code.initialize(generated_pygments_data);
     message_edit_history.initialize();
+    if (recent_topics_util.is_visible()) {
+        // If the page is loaded to directly to recent_topics
+        // either as loading to "#recent_topics" or by loading the
+        // default view when the default view is recent_topics
+        // then we make this call to bind our handler.
+        recent_topics_ui.initialize_handler();
+    }
```

Comment:
We aim to have no conditionals in this initialization function; you can put that if statement in `recent_topics_ui` (or whatever module here).  

For similar convention reasons, we should call this e.g.`recent_topics_ui.initialize()` if at all possible.